--- apertium-oc-es-1.0.5/Makefile.am.orig	2008-07-21 14:53:50.000000000 +0200
+++ apertium-oc-es-1.0.5/Makefile.am	2010-12-31 13:18:14.942122774 +0100
@@ -267,9 +267,9 @@
 
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
-	$(INSTALL_DATA) oc-es.mode $(apertium_oc_esmodesdir)
-	$(INSTALL_DATA) es-oc.mode $(apertium_oc_esmodesdir)
-	$(INSTALL_DATA) oc_aran-es.mode $(apertium_oc_esmodesdir)
-	$(INSTALL_DATA) es-oc_aran.mode $(apertium_oc_esmodesdir)
+	$(INSTALL_DATA) oc-es.mode $(DESTDIR)$(apertium_oc_esmodesdir)
+	$(INSTALL_DATA) es-oc.mode $(DESTDIR)$(apertium_oc_esmodesdir)
+	$(INSTALL_DATA) oc_aran-es.mode $(DESTDIR)$(apertium_oc_esmodesdir)
+	$(INSTALL_DATA) es-oc_aran.mode $(DESTDIR)$(apertium_oc_esmodesdir)
 	
 CLEANFILES = -r $(TARGETS)
